hack/env.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Copyright 2020 Red Hat, Inc. and/or its affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DEPLOY_DIR="deploy"
+OLM_DIR="${DEPLOY_DIR}/olm-catalog/kogito-operator"
```

Comment:
Where is this used?, we don't have `OLM_DIR` any more

Reply:
rebase problem ... updated ;)